android/src/main/README.md

```diff
@@ -0,0 +1,6 @@
+# Placeholder directory for generated Kotlin bindings
+
+Expecting to see some `.kt` source files in this directory?
+Don't worry, you're in the right place! It's just that all our
+Kotlin files are generated automatically as part of the gradle
+build process, using `uniffi-bindgen`.
```

Comment:
The empty directory here briefly alarmed me until I remembered that all the hand-written Kotlin was going to be in a-c 😆